ADAL/tests/XCTestCase+TestHelperMethods.m

```diff
@@ -501,5 +496,136 @@ - (ADTestURLResponse *)adResponseAuthCode:(NSString *)authCode
     return response;
 }
 
+- (MSIDTokenCacheItem *)adCreateAccessMSIDTokenCacheItem
+{
+    MSIDTokenCacheItem *tokenCacheItem = [MSIDTokenCacheItem new];
+    tokenCacheItem.accessToken = TEST_ACCESS_TOKEN_TYPE;
+    tokenCacheItem.refreshToken = nil;
+    tokenCacheItem.idToken = [self adCreateUserInformation:TEST_USER_ID].rawIdToken;
+    tokenCacheItem.expiresOn = [NSDate dateWithTimeIntervalSince1970:1500000000];
+    tokenCacheItem.cachedAt = nil;
+    tokenCacheItem.familyId = nil;
+    tokenCacheItem.clientInfo = [self adCreateClientInfo];
```

Comment:
same comment about using MSID utilities here.

Reply:
I didn't find client info util in msid